### PR TITLE
x11-libs/vte: Version bump to 0.70.4

### PR DESCRIPTION
**Pull request by listout**

### CODE REVIEW

**Comment by listout** [comment]
Once #30429, I can then rebase this

**Comment by mattst88** [comment]
Please handle `gui-libs/vte` at the same time.

**Comment by listout** [comment]
I'm doing so, do I create a separate commit for it?

**Comment by listout** [comment]
> Please handle `gui-libs/vte` at the same time.

Thanks, done :)

**Comment by mattst88** [comment]
> x11-libs/vte: gui-libs/vte-common: gui-libs/vte-common: Version bump to 0.70.4

One commit per package, please.

**Comment by listout** [comment]
> > x11-libs/vte: gui-libs/vte-common: gui-libs/vte-common: Version bump to 0.70.4
> 
> One commit per package, please.

Done

File: x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Build fails with New clang 16.0 compiler because it finds a new error -Wenum-constexpr-conversion
+# Upstream issue: https://gitlab.gnome.org/GNOME/vte/-/issues/2618
+# Patch authored by Khem Raj
```
Comment (thesamesam) [inline x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch:3]: Please always put the commit link in the patch too.

Also, to preserve metadata, append .patch to the URL to the commit and use that as the basis for the patch.

Comment (listout) [reply x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch:3]: Doing it.

Comment (thesamesam) [reply x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch:4]: It's got the link but you haven't included all the metadata at the top?

Comment (listout) [reply x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch]: My bad, is it alright now?

File: x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# From 9400d41660e6c27b672f9c77855d736581f499bf Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch:1]: Thanks, lgtm now. I personally put our own notes above the "From" line (so it doesn't look like they wrote it), but that's personal taste.

Cheers!

Comment (listout) [reply x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch:1]: Nah, lets keep it that way, after all Khem is the original author.

**Comment by mattst88** [comment]
>  x11-libs/vte/vte-0.70.4-r1.ebuild | 105 ++++++++++++++++++++++++++++++
>  x11-libs/vte/vte-0.70.4.ebuild    | 101 ++++++++++++++++++++++++++++

I don't think there's any need for this?

>  x11-libs/vte/vte-0.70.3-r1.ebuild             | 105 ++++++++++++++++++
>  x11-libs/vte/vte-0.70.3.ebuild                |   4 +

I also don't think there's any need to touch 0.70.3, and even less if you're making a 0.70.3-r1.

**Comment by listout** [comment]
> 

Fixed

**Pull request closed without merging**